7.figures/nbconverted/Figure3_model_evaluation.r

```diff
@@ -34,18 +38,31 @@ cm_df$Predicted_Label <-
 
 # Order feature types for plotting
 cm_df$feature_type <-
-    dplyr::recode_factor(cm_df$feature_type, !!!facet_labels)
+    dplyr::recode_factor(cm_df$feature_type, !!!feature_spaces)
 
+print(dim(cm_df))
 head(cm_df)
 
+focus_feature_spaces <- c(
```

Comment:
What is the reason you set these here, but you have feature spaces also set in the `themes.r` file?

I am assuming it is because these variables are used in different ways, but still wanted to ask :smile:

Reply:
This is a great question! Your assumption is indeed correct, but I want to write code in such a way as to avoid the need to make assumptions!

I will make this more clear in an upcoming commit. Thanks!

Reply:
Fixed in d1b552fabbe29aceb08307fab5dd55df9169b119